lib/compat/wordpress-6.8/functions.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * Adds the x-wav mime type to the list of mime types.
+ * This is necessary for Firefox as it uses the identifier for uploaded .wav files.
+ * Core backport should add the following to the default mime_types filters in
+ * `wp_get_mime_types()` in wp-includes/functions.php:
+ *
+ * `'wav|x-wav' => 'audio/wav'`
+ *
+ * @since 6.8.0
+ *
+ * @param string[] $mime_types Mime types.
+ * @return string[] Mime types keyed by the file extension regex corresponding to those types.
+*/
+function gutenberg_get_mime_types_6_8( $mime_types ) {
+	/*
+	 * Only add support if there is existing support for 'wav'.
+	 * Some plugins may have deliberately disabled it.
+	*/
+	if ( ! $mime_types['wav'] && ! isset( $mime_types['wav|x-wav'] ) ) {
```

Comment:
Absolute Nitpick: We already have the `! isset( $mime_types['wav|x-wav'] )` check in line 30, do we also need it here?

Or maybe we can simplify it like this?

```
if ($mime_types['wav'] && !isset($mime_types['x-wav']) && !isset($mime_types['wav|x-wav'])) {
        $mime_types['x-wav'] = 'audio/wav';
    }
```

But no big deal, the code already looks good 👍

Reply:
Thanks for reviewing!! 

> Or maybe we can simplify it like this?

You're right, and that's how I had it originally, however I split it up to make it easier to read, and for my small brain to process. 

So, condition 1 says "If wav AND wav|x-wav are not there we assume it's a deliberate measure by the theme/plugin and return early"

Condition 2 just checks if x-wav is absent.